### PR TITLE
Adding more debug to wlp.lib.extract.PackageRunnableTest.testRunnableJar

### LINKED ISSUE #14982: Add debug to wlp.lib.extract.PackageRunnableTest.testRunnableJar to dump manifest contents on failure
The wlp.lib.extract.PackageRunnableTest.testRunnableJar test fails intermittently with an error regarding the manifest having a problem.  Adding code to dump the contents of the manifest in the failure case for troubleshooting.

### PR DESCRIPTION
**Pull request by hlhoots**
fixes #14982 
#build

### CODE REVIEW

File: dev/wlp.lib.extract_fat/fat/src/wlp/lib/extract/PackageRunnableTest.java [hunk 1]
```diff
@@ -297,7 +299,25 @@ private String executeTheJar(File extractDirectory, boolean useDummyUserDir, boo
             } else {
                 Log.info(c, "executeTheJar", "No messages.log - " + messagesLog.getAbsolutePath());
             }
+
```
Comment (tjwatson) [inline dev/wlp.lib.extract_fat/fat/src/wlp/lib/extract/PackageRunnableTest.java:302]: I suggest just using a BufferedReader with the `META-INF/MANIFEST.MF` JarEntry input stream to log it line by line.  I worry that the JarFile Manifest object may give us an empty  attributes even when the manifest resource has some corrupted data in it.

Comment (hlhoots) [reply dev/wlp.lib.extract_fat/fat/src/wlp/lib/extract/PackageRunnableTest.java:299]: Thanks Tom!  Updates made based on your suggestion.

**Comment by hlhoots** [comment]
#build

**Comment by hlhoots** [comment]
#build

**Review by tjwatson — APPROVED** [review]

**Comment by hlhoots** [comment]
#libby

**Pull request merged**